src/Switchboard.vala

```diff
@@ -309,6 +322,27 @@ namespace Switchboard {
 
             add_window (main_window);
 
+            navigation_button.clicked.connect (handle_navigation_button_clicked);
+
+            search_box.changed.connect(() => {
+                category_view.filter_plugs(search_box.get_text ());
```

Comment:
Need some spaces before brackets on the two lines above here. I know this was in the original, but may as well fix while we're tidying.